front/app/api/ideas/types.ts

```diff
@@ -137,8 +137,8 @@ export interface IIdeaData {
 export interface IIdeaAdd {
   // Required
   project_id: string;
-  publication_status: IdeaPublicationStatus;
   // Optional
+  publication_status?: IdeaPublicationStatus;
```

Comment:
Because the when we post an idea, the publication status was always published but I got an error now with the pre-screening flow where it didn't work. So I made it optional and I didn't pass it anymore where it started to work as expected. I am happy to make it required again, if you resolve the error on the back-end side.

Reply:
@sebastienhoorens Do you think you can resolve this in this PR already? Would be most efficient, I think.

Reply:
@IvaKop Yes, we can merge it like this, I was just wondering. But if this is needed to post an idea without publication status, then your change probably makes sense.

How it works in the backend:
- When posting an input the publication_status is not required, and the backend will choose the right default value (published or submitted) depending if reviewing is enabled (in prescreening) or not (in proposed).
- But at any time, all inputs in the database will have a value for the publication_status, so in that sense it is always required in the backend. But perhaps these requirements work differently on the frontend side?

Reply:
Ok, so in that case it should work correctly still. On the front-end the optional parameter is only for the post request. It is always required in the response which seems to be correct.